.bundler-audit.yml

```diff
@@ -1,2 +1,4 @@
 ignore:
   - CVE-2015-9284
+  - CVE-2024-26142
+  - CVE-2024-26143
```

Comment:
Changes made here, actioncable > 7.1.1 has issues not resolved in further versions at this time